src/renderers/WebGLRenderer.js

```diff
@@ -1339,13 +1339,27 @@ class WebGLRenderer {
 
 			const isWebGL2 = capabilities.isWebGL2;
 
+			const currentRenderTarget = _this.getRenderTarget();
+
+			// If the current render target has a stencil buffer, then the transmission render target also needs it so that it's rendered the same way.
+
+			const needStencilBuffer = currentRenderTarget !== null ? currentRenderTarget.stencilBuffer : _this.getContextAttributes().stencil;
```

Comment:
The name `needsStencilBuffer` would be more consistent to other `needs*` flags.

Reply:
Fixed!